packages/paste-core/components/avatar/src/utils.ts

```diff
@@ -45,35 +45,46 @@ export const getCorrespondingFontSizeFromSizeToken = (size: IconSize): FontSizeO
   }
 };
 
-export const getCorrespondingIconSizeFromSizeToken = (size: IconSize): IconSizeOptions => {
-  switch (size) {
-    case 'sizeIcon10':
-    case 'sizeIcon20':
-    case 'sizeIcon30':
-    case 'sizeIcon40':
-    default:
-      return 'sizeIcon10';
-    case 'sizeIcon50':
-    case 'sizeIcon60':
-      return 'sizeIcon20';
-    case 'sizeIcon70':
-      return 'sizeIcon30';
-    case 'sizeIcon80':
-      return 'sizeIcon40';
-    case 'sizeIcon90':
-      return 'sizeIcon50';
-    case 'sizeIcon100':
-      return 'sizeIcon70';
-    case 'sizeIcon110':
-      return 'sizeIcon80';
+export const getCorrespondingIconSizeFromSizeToken = (size: IconSizeOptions): IconSizeOptions => {
+  if (typeof size === 'string' && size.includes('sizeIcon')) {
```

Comment:
Should we add this check into `getCorrespondingFontSizeFromSizeToken` and `getComputedTokenNames` (line 80) as well?

Reply:
We added this check to `getCorrespondingFontSizeFromSizeToken` and `getCorrespondingLineHeightFromSizeToken`, but we're wondering if it's necessary to add the check to `getComputedTokenNames` as well (the function that calls `getCorrespondingFontSizeFromSizeToken`, etc.). The checks will already run every time the function is called, so is it necessary to duplicate them within `getComputedTokenNames`?

Reply:
> The checks will already run every time the function is called, so is it necessary to duplicate them within getComputedTokenNames

I think your logic is sound there. No need to add it to `getComputedTokenNames`